en/components/grids_templates/editing.md

```diff
@@ -152,9 +158,23 @@ Another way to update cell is directly through [`update`]({environment:angularAp
 ```
 }
 
-@@if (igxName === 'IgxGrid') {
-If you want to define a custom template which will be applied when the cell is in edit mode, you can see the documentation for [Grid Columns configuration](grid.md#columns-configuration).
-}
+#### Cell Editing Templates
+
+If you want to provide a custom template which will be applied when a cell is in edit mode, you can make use of the [`igxCellEditor` directive]({environment:angularApiUrl}/classes/igxcelltemplatedirective.html). In order to use a custom cell editing template for a column, you need to pass an `ng-template` with the `igxCellEditor` directive and properly bind your custom control to the [`cell.editValue`]({environment:angularApiUrl}/classes/igxcellcomponent.html#editvalue):
+
+```html
+<igx-column field="ReorderLevel" header="Reorder Level" [editable]="true">
+    <ng-template igxCellEditor let-cell="cell">
+        <input type="number" [(ngModel)]="cell.editValue" />
+    </ng-template>
+</igx-column>
+```
+
+> [!NOTE]
+> The cell editing template directive (`igxCellEditor`) is very different from the [cell template(`igxCell`)](../grid/grid.md#cell-template) - the former handles how a column's cells **in edit mode** are displayed an controls the edited cell's **edit value**, the latter control how a column's cell's are show when **outside of edit mode**.
```

Comment:
@rkaraivanov , could you take a look when you've got the time?
Does this make a clear enough difference between **cell template** and **cell editing template**, or should I add some extra explanation?

Reply:
@ViktorSlavov 

IMO, I think the main distinction should be between the functional differences between the two directives. In the end, they both allow for customizing the presentation of cells. Thus an user can still re-template the default cell content through the `igxCell` directive and achieve CRUD functionality even when the column may not be editable. For example:

```html
<ng-template igxCell let-value>
     <!-- This will update the underlying record even when the column is not editable -->
    <input [(ngModel)]="value" /> 
</ng-template>
```

What he won't get however are all the CRUD events, transaction support, etc. I think that should be the focus point of the documentation, backed up by some code snippets/samples which actually show the difference.